assets/javascript/app.js

```diff
@@ -50,12 +50,20 @@ $(document).ready(function() {
             var state = $('#address-state').val()
             var zip = $('#address-zip').val()
 
-
-            if (address1 !== '' && (city === '' && state === '' && zip === '')) {
+            console.log(distance.toString())
+            console.log($('#distance').val())
+            if (address1 === '' && city === '' && state === '' && zip === '' && distance.toString() === $('#distance').val()) { 
```

Comment:
Added error messaging if user attempts to submit a change that does not change the outcome of the current results.